src/FormButton/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Omit } from '@stoplight/types';
+import _noop = require('lodash/noop');
+import * as React from 'react';
+// TODO: should probably use ajv and json schema
+import * as yup from 'yup';
+
+import { Button, IButtonProps } from '../index';
+
+/**
+ * FORM BUTTON
+ * button that is disabled if the schema doesn't pass validation
+ * TODO, add tooltip to explain why its disabled?
```

Comment:
Could make it optional but meh..

Reply:
will add when needed, I think with a combination of form inputs we can do without